mainnet/2024-03-05-pause-unpause-test/Makefile

```
@@ -15,27 +15,27 @@ endif
 # Pause OptimismPortal Commands
 
 .PHONY: pause-portal-sign
-pause-portal-sign: deps
+pause-portal-sign:
 	$(GOPATH)/bin/eip712sign --ledger --hd-paths "m/44'/60'/$(LEDGER_ACCOUNT)'/0/0" -- \
 	forge script --rpc-url $(L1_RPC_URL) PausePortal --sig "sign()"
 
 .PHONY: pause-portal-run
-pause-portal-run: deps
-	forge script --rpc-url $(L1_RPC_URL) \
-	PausePortal --sig "run(bytes)" $(SIGNATURES) \
-	--ledger --hd-paths "m/44'/60'/$(LEDGER_ACCOUNT)'/0/0" \
+pause-portal-run:
+	@forge script --rpc-url $(L1_RPC_URL) \
+	PausePortal --sig "run(bytes)" $(PAUSE_SIGNATURES) \
+	--private-key $(EXECUTOR_PRIVATE_KEY) \
```

This is a safer pattern to use for executing calls against mainnet. In this way, funded pkeys don't need to be put into local .env files. Are you planning on executing this with a key that isn't on your ledger?  
```suggestion
	--ledger --hd-paths "m/44'/60'/$(LEDGER_ACCOUNT)'/0/0" \
```